app/assets/javascripts/global.js

```diff
@@ -42,7 +41,13 @@ function toggle_loading(){
 //Toggle a forms submit button when a checkbox is checked/unchecked
 function accept_and_enable(box)
 {
-    var submit = document.getElementById("TOC");
+    //var submit = document.getElementById("TOC");
+
+
+    var submit = $(box).parent().parent().parent().parent().find("input[type='submit']")[0];
```

Comment:
Did you (by any chance) find a better way to handle this? Maybe `parent_until` wth an additional `parent`, or `closest`?